src/ui/components/job-saved-success/job-saved-success.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Col, Row } from 'antd';
+import { WEB_CLIENT_URL } from '../../../config/config';
+
+interface ISuccessData {
+  company: string,
+  role: string,
+  id: string
+}
+
+interface IJobSavedSuccess {
+  data: ISuccessData
+}
+
+const JobSavedSuccess = (props: IJobSavedSuccess) => {
+  const { data } = props;
+  return (
+    <div className="success-message">
+        <Row>
+          <Col span={6}>
+            <img src={chrome.runtime.getURL('images/teal_logo_43.svg')}/>
+          </Col>
+          { 
+            data.id && <Col span={18}>
```

Comment:
nitpicky formatting thing. This is easier to read when formatted like so:

```suggestion
          { data.id &&
              <Col span={18}>
```